packages/@okta/vuepress-site/docs/guides/build-sso-integration/main/openidconnect/prep.md

```diff
@@ -1,33 +1,150 @@
 
 For example, consider a scenario where your app integration is added to 10 separate customer orgs. Seven of those customers create a single instance of your app integration. However, the other three customers each create two separate instances of your app integration so they can use different configuration options. This scenario creates a total of 13 sets of client credentials for your application that you need to track.
 
-This multi-tenant approach is different from other IdPs that use a global credential system, where a given application has the same customer credentials across all orgs.
+This multi-tenant approach differs from other IdPs that use a global credential system, where a given application has the same customer credentials across all orgs.
+
+### Determine the OAuth 2.0 flow to use
+
+Select the OAuth 2.0 flow to use based on your app:
+
+* For web apps:
+
+   Okta recommends the [Authorization Code flow](/docs/guides/implement-grant-type/authcode/main/). This flow is used for apps with a dedicated server-side back-end capable of securely storing a client secret. The app integration can also exchanging information with an authorization server through trusted back-channel connections.
+
+* For single-page apps (SPA):
+
+   Okta recommends the [Authorization Code flow with a Proof Key for Code Exchange (PKCE)](/docs/concepts/oauth-openid/#authorization-code-flow-with-pkce) to control access between your SPA application and a resource server.
+
+> **Note:** Native and mobile app integrations aren't accepted as OIDC app integrations in the OIN. Set up your app to use an authentication flow that allows your client app to talk to your SaaS back end. Your SaaS back end can then securely communicate with Okta through trusted back-channel connections.
+
+Follow these guides to implement the OAuth 2.0 flows:
+
+* [Implement Authorization Code flow](/docs/guides/implement-grant-type/authcode/main/)
+* [Implement Authorization Code flow with PKCE](/docs/guides/implement-grant-type/authcodepkce/main/)
+
+> **Note:** You can also review these sample app integration quickstarts that use the Okta CLI:
+> * [Sign in to SPA](/docs/guides/sign-into-spa-redirect/)
+> * [Sign in to web application](/docs/guides/sign-into-web-app-redirect/)
+
+When you follow these guides, one caveat is the use of the authorization server. Most of the examples show you how to make an `/authorize` or `/token` request using a [custom authorization server](/docs/concepts/auth-servers/#custom-authorization-server). To support the potentially large numbers of Okta orgs accessing it through the OIN, an OIDC integration can't use a custom authorization server, including the `default` server. You can only use the [org authorization server](/docs/concepts/auth-servers/#org-authorization-server).
```

Comment:
If an integration in OIN is installed, then a customer wants to do a jwt service call with OIDC integration with custom scopes, I don't think it's possible to do so with the org authorization server. Is this just a known limitation with OIN? 

Or is it that when the end customer installs it into their org, they can then use a custom authorization server, but we just don't want ISV to assume they can use custom/default auth server that's not org server?

 I haven't tested this use case myself so Im more curious which of the 2 behaviours happen